sdk/analytics/src/main/java/com/klaviyo/analytics/networking/KlaviyoApiClient.kt

```diff
@@ -32,17 +33,22 @@ internal object KlaviyoApiClient : ApiClient {
     private var apiObservers = mutableListOf<ApiObserver>()
 
     init {
-        onApiRequest {
-            Registry.log.info(
-                when (it.state) {
-                    KlaviyoApiRequest.Status.Unsent.name -> "Request Enqueued"
-                    KlaviyoApiRequest.Status.PendingRetry.name -> "Request will be retried"
-                    KlaviyoApiRequest.Status.Complete.name -> "Request Completed"
-                    else -> "Request Failed"
-                }
-            )
+        onApiRequest { r ->
```

Comment:
So before this was logging one thing but now it's logging a number of things. I'm a little out of the loop but are you using this logging to drive the test app. Is there some way you re-assemble this on the other end?

Reply:
This is _just_ debug logging, the test app can subscribe using the same `onApiRequest` callback mechanism that this block is using.

This particular change is just fixing a mistake I'd made before. Earlier, I just logged a message like `Request Queued` and then logged `Request.toString()`, but that toString wasn't doing what I expected. Decided to just make it much more explicit so the output logs are nice and readable. Also changed it to log level verbose so it can be filtered out more easily